ui/ui-components/components/layouts/Main.tsx

```diff
@@ -171,7 +171,7 @@ export default function Main(props) {
           <div
             style={{
               position: "fixed",
-              top: 0,
+              bottom: 0,
```

Comment:
Instead of this, it looks like a Toast.Container can handle it for us. Unless we're avoiding that for some specific reason?
https://react-bootstrap.github.io/components/toasts/#toast-container-props

Reply:
I don't think that exists in react-bootsrap v1 (bootstrap 4) which we are still using in core

Reply:
wait! It does! https://react-bootstrap.netlify.app/components/toasts/#toast-container-props

Reply:
NVM - ToastConatiner is not in the version we are using https://react-bootstrap-v4.netlify.app/components/toasts/